src/html/partials/header.html

```diff
@@ -1,18 +1,25 @@
 <header class="header">
 
-  <div class="wrapper--title">
+  <div class="header-wrapper">
+
+    <div class="header__background-img img-1"></div>
```

Comment:
thanks. since this could be applied in other places, i'm putting this under my "[nice to haves](https://github.com/prechter/florence/projects/1)" if there's time at the end. rn i'm a little behind with styling, but if there's time i'll try to take a pass on all the markup and use nunjucks a little more properly